fe2-android/app/src/test/java/com/github/dedis/student20_pop/model/LaoTest.java

```diff
@@ -1,172 +1,253 @@
-  //  public void setNullWitnessesTest() {
-  //    assertThrows(IllegalArgumentException.class, () -> LAO_1.setWitnesses(null));
-  //    assertThrows(IllegalArgumentException.class, () -> LAO_1.setWitnesses(WITNESSES_WITH_NULL));
-  //  }
-  //
-  //  @Test
-  //  public void setNullMembersTest() {
-  //    assertThrows(IllegalArgumentException.class, () -> LAO_1.setMembers(null));
-  //    assertThrows(IllegalArgumentException.class, () -> LAO_1.setMembers(WITNESSES_WITH_NULL));
-  //  }
-  //
-  //  @Test
-  //  public void setNullEventsTest() {
-  //    assertThrows(IllegalArgumentException.class, () -> LAO_1.addEvent(null));
-  //    assertThrows(IllegalArgumentException.class, () -> LAO_1.setEvents(null));
-  //    assertThrows(IllegalArgumentException.class, () -> LAO_1.setEvents(EVENTS_WITH_NULL));
-  //  }
-  //
-  //  @Test
-  //  public void getNullIdsTest() {
-  //    assertThrows(IllegalArgumentException.class, () -> Lao.getIds(null));
-  //    assertThrows(IllegalArgumentException.class, () -> Lao.getIds(LAOS_WITH_NULL));
-  //  }
-  //
-  //  @Test
-  //  public void getIdsTest() {
-  //    assertThat(
-  //        Lao.getIds(LAOS),
-  //        is(new ArrayList<>(Arrays.asList(LAO_1.getId(), LAO_FOR_EVENTS.getId()))));
-  //  }
-  //
-  //  @Test
-  //  public void equalsTest() {
-  //    Lao lao1 = new Lao(LAO_NAME_1, ORGANIZER, HOST);
-  //    Lao lao2 = new Lao(LAO_NAME_1, ORGANIZER, HOST);
-  //    Lao lao3 = new Lao(LAO_NAME_2, ORGANIZER, HOST);
-  //
-  //    assertEquals(lao1, lao2);
-  //    assertNotEquals(lao1, lao3);
-  //  }
-  //
-  //  @Test
-  //  public void hashCodeTest() {
-  //    Lao lao1 = new Lao(LAO_NAME_1, ORGANIZER, HOST);
-  //    Lao lao2 = new Lao(LAO_NAME_1, ORGANIZER, HOST);
-  //    Lao lao3 = new Lao(LAO_NAME_2, ORGANIZER, HOST);
-  //
-  //    assertEquals(lao1.hashCode(), lao2.hashCode());
-  //    assertNotEquals(lao1.hashCode(), lao3.hashCode());
-  //  }
+    private static final String LAO_1_ID = "lao1Id";
+    private static final String LAO_NAME_1 = "LAO name 1";
+    private static final String ORGANIZER = "0x2365";
+    private static final String rollCallId1 = "rollCallId1";
+    private static final String rollCallId2 = "rollCallId2";
+    private static final String rollCallId3 = "rollCallId3";
+    private static final String electionId1 = "electionId1";
+    private static final String electionId2 = "rollCallId2";
+    private static final String electionId3 = "rollCallId3";
+    private static final Set<String> WITNESSES = new HashSet<>(Arrays.asList("0x3434", "0x4747"));
+    private static final Set<String> WITNESSES_WITH_NULL = new HashSet<>(Arrays.asList("0x3939", null, "0x4747"));
+
+    private static final Lao LAO_1 = new Lao(LAO_1_ID, LAO_NAME_1);
+    private static final Map<String, RollCall> rollCalls = new HashMap<String, RollCall>() {{
+        put(rollCallId1, new RollCall());
+        put(rollCallId2, new RollCall());
+        put(rollCallId3, new RollCall());
```

Comment:
The class RollCall doesn't have a default constructor, either implement one or create a roll call using the existing constructor. You have this issue multiple times on this file

Reply:
I don't really understand why you mean by that, isn't this the default constructor in RollCall that I use  : public RollCall() {
    this.attendees = new HashSet<>();
  }
?

Reply:
I see, you have the old RollCall constructor, you should merge with master